remsfal-core/pom.xml

```diff
@@ -55,6 +55,16 @@
             <artifactId>microprofile-openapi-api</artifactId>
             <version>${version.open-api}</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+            <version>3.13.3</version>
+        </dependency>
+        <dependency>
```

Comment:
Move this dependency to /remsfal-service/pom.xml

If you need annotations like @PartType or @MultipartForm, please use the following dependency..

https://mvnrepository.com/artifact/org.jboss.resteasy/resteasy-multipart-provider/6.2.10.Final

<!-- https://mvnrepository.com/artifact/org.jboss.resteasy/resteasy-multipart-provider -->
<dependency>
    <groupId>org.jboss.resteasy</groupId>
    <artifactId>resteasy-multipart-provider</artifactId>
    <version>6.2.10.Final</version>
</dependency>